spool.py

```diff
@@ -121,13 +121,13 @@ def spool_render(rman_version_short, rib_files, denoise_files, frame_begin, fram
                                     [('PixarRender', cmd_str)], 3)
             else:
                 denoise_options.append('-F 1 -L 1')
-                cmd_str = ['denoise'] + denoise_options + [f[0] for f in denoise_files[frame_num - 3: frame_num]]
+                cmd_str = ['denoise'] + denoise_options + [f[0] for f in denoise_files[frame_num - frame_begin - 2: frame_num - frame_begin + 1]]
                 write_cmd_task_line(f, 'Denoise frame %d' % (frame_num - 1),
                                     [('PixarRender', cmd_str)], 3)
             if frame_num == frame_end:
-                denoise_options.remove('-F 1 -L 1')
+                denoise_options = (x for x in denoise_options if x not in  ['-L 1', '-F 1 -L 1'])
```

Comment:
did you want denoise_options to be a list or tuple?  List seems more appropriate